src/screenipy.py

```diff
@@ -79,14 +85,50 @@ def initExecution():
         Utility.tools.clearScreen()
         return initExecution()
 
+
+def isTradingTime():
+    curr = datetime.now(pytz.timezone('Asia/Kolkata'))
+    openTime = curr.replace(hour=9, minute=15)
+    closeTime = curr.replace(hour=15, minute=30)
+    isTradingTime = ((openTime <= curr <= closeTime) and (
+        0 <= curr.weekday() <= 4))
+
+
+def saveStockData():
+    global stockDict
+    today_date = date.today().strftime("%d%m%y")
+    cache_file = "stock_data_" + str(today_date) + ".pkl"
+    if not os.path.exists(cache_file):
+        with open(cache_file, 'wb') as f:
+            pickle.dump(stockDict.copy(), f)
+
+
+def loadStockData(stockDict):
+    today_date = date.today().strftime("%d%m%y")
+    cache_file = "stock_data_" + str(today_date) + ".pkl"
+    if os.path.exists(cache_file):
+        with open(cache_file, 'rb') as f:
+            stockData = pickle.load(f)
+            for stock in stockData:
+                stockDict[stock] = stockData.get(stock)
+
```

Comment:
Let's keep main code clean by moving this methods to the `Utility` class.

Reply:
Ok..